src/backend/distributed/commands/owned.c

```diff
@@ -88,3 +88,36 @@ PreprocessDropOwnedStmt(Node *node, const char *queryString,
 
 	return NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
 }
+
+
+List *
+PreprocessReassignOwnedStmt(Node *node, const char *queryString,
+							ProcessUtilityContext processUtilityContext)
+{
+	ReassignOwnedStmt *stmt = castNode(ReassignOwnedStmt, node);
+	List *allReassignRoles = stmt->roles;
+
+	List *distributedReassignRoles = FilterDistributedRoles(allReassignRoles);
```

Comment:
.

Reply:
I could not understand what dot means?

Reply:
Wanted to point out which portion of the code https://github.com/citusdata/citus/pull/7319#pullrequestreview-1715969930 is talking about, nvm :)